app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    skills = []
+
```

Comment:
skills should be inside constructor

Reply:
Fix it. Skills should be the instance's attribute